web_generator/test/ts_bindings_test.dart

```diff
@@ -0,0 +1,75 @@
+@TestOn('vm')
+@Tags(['node'])
+
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:test/test.dart';
+
+import 'utils/run_process.dart';
+
+void main() {
+  final testGenFolder = p.join('test', 'gen');
+  final testGenDTSFiles = p.join(testGenFolder, 'input');
+
+  final bindingsGenPath = p.join('lib', 'src');
+
+  group('Web Generator TS Bindings Integration Test', () {
+    final inputDir = Directory(testGenDTSFiles);
+
+    setUp(() async {
+      // set up npm
+      await runProcess(
```

Comment:
With the above comment to just refactor the `runProcess` calls, we could just use those `cli.dart` functions directly here instead of this.